frontend/index.mjs

```diff
@@ -26,6 +21,7 @@ export default async function (plop) {
       },
     ],
     actions: function (data) {
+      data.criName = plop.getDestBasePath().split("/").slice(-1)[0];
```

Comment:
Why not? `data.criName = path.basename(plop.getDestBasePath());`

Reply:
Do we want to remove this?

Reply:
I think that might work, I'll test and update in the next PR if it does